src/shared/components/RichTable.tsx

```diff
@@ -117,7 +117,7 @@ export interface RichTableRow<D extends object> {
 
 export interface RichTableHook<D extends object = {}> {
   columns: Column<D>[];
-  url: string;
+  get: (args: RequestArgs) => Promise<APIResponse<D | Pagination<D>, APIError>>;
```

Comment:
Second type argument to APIResponse is unneeded

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/9aa2bdd2f117930cdf0a9ce0dedbf1f7c6d186ac